proprietary/Components/src/denhaag/descriptionlist.tokens.json

```diff
@@ -0,0 +1,123 @@
+{
+  "denhaag": {
+    "descriptionlist": {
+      "border-color": {
+        "value": "{denhaag.color.grey.2}"
+      },
+      "color": {
+        "value": "{denhaag.color.grey.4}"
+      },
+      "font": {
+        "family": {
+          "value": "{denhaag.typography.sans-serif.font-family}"
+        },
+        "size": {
+          "value": "1.125rem"
```

Comment:
This is actually the new base font size. (18px). However, I think we should modify the `denhaag.typography.scale.base` token in a separate PR since it affects so many components.